README.md

```diff
@@ -1,3 +1,37 @@
 # Grunt incuna plugins
 
 A set of useful build and testing plugins easily reusable across projects.
+
+* [Adding new plugins](#adding-new-plugins)
+* [Plugins](#plugins)
+ * [update-po-files](#update-po-files)
+
+## Plugins
+
+### `update-po-files`
+
+A plugin for automatically updating `.po` files in a given directory from a `template.pot` file in that same directory.
+Usage:
+```js
+'update-po-files': {
+    all: {
+        src: 'my-i18n-directory'
```

Comment:
you'd also need a `templatePath` property too to locate the `.pot` file though.
But Its a lot more flexible and follows the standard grunt conventions